test/unit/container_image_oci_test.py

```diff
@@ -164,17 +167,16 @@ def test_create_derived(
 
         mock_create.assert_called_once_with('kiwi_oci_dir.XXXX/oci_layout')
 
-        self.oci.oci.init_layout.assert_called_once_with(
-            'root_dir/image/image_file'
-        )
+        self.oci.oci.init_layout.assert_called_once_with(True)
         self.oci.oci.unpack.assert_called_once_with('kiwi_oci_root_dir')
         self.oci.oci.repack.assert_called_once_with('kiwi_oci_root_dir')
         self.oci.oci.set_config.assert_called_once_with({
             'container_name': 'foo/bar',
             'additional_tags': ['current', 'foobar'],
             'container_tag': 'latest',
-            'entry_subcommand': ['/bin/bash']
-        })
+            'entry_subcommand': ['/bin/bash'],
+            'history': {'created_by': 'KIWI {0}'.format(__version__)}
+        }, True)
```

Comment:
I had added the `base_image` as real Boolean because this test was really confusing if not. I know the result is the same, but having a string pointing to a directory here felt really strange.

Reply:
yes understood, actually I also liked what @cyphar suggested

```python
bool(base_image)
```